### ISSUE TITLE
Malformed manifest with `permissions` field.

### ISSUE THREAD

**Issue by hurui200320** [neow3j/neow3j#631]
Hi, I noticed a malformed manifest generated by the compiler.

```Json
  "permissions" : [ {
    "contract" : "*",
    "methods" : "*"
  } ],
```

The `methods` field can be `["*"]` it `"*"`, both of them are ok to Neo-GUI and neo node, but only `["methodName"]` is accepted by Neo-GUI and neo node, the `"methodName"` will be rejected by Neo-GUI and neo node will give error: `One of the identified items was in an invalid format.`

**Comment by hurui200320** [comment]
The manifest file is generated by the Gradle plugin, and the error is given when deploying the contract by code.

**Comment by csmuller** [comment]
Solved in #630.
